oauth2/src/main/java/com/okta/spring/boot/oauth/ReactiveOktaOAuth2AutoConfig.java

```diff
@@ -69,13 +64,13 @@ OidcReactiveOAuth2UserService oidcUserService(Collection<AuthoritiesProvider> au
     }
 
     @Bean
-    @ConditionalOnBean(ReactiveJwtDecoder.class)
```

Comment:
Removing this was necessary to invoke this configuration when no `SecurityConfiguration` exists.